src/aioquic/quic/connection.py

```diff
@@ -304,6 +304,8 @@ def __init__(
             frame_type=QuicFrameType.MAX_STREAMS_UNI, name="max_streams_uni", value=128
         )
         self._loss_at: Optional[float] = None
+        assert configuration.max_packet_size >= 1200, "The smallest allowed maximum packet size is 1200 bytes"
```

Comment:
Can we move this assertion up to the top of the constructor with the other ones?